packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -3585,6 +3585,13 @@ body {
         imageSizes: 'makes no sense',
       });
 
+      ReactDOM.preload('rp', {
+        as: 'image',
+        imageSrcSet: 'rpsrcset',
+        imageSizes: 'rpsizes',
+        referrerPolicy: 'no-referrer',
+      });
+
```

Comment:
Can we just make a new test. The preload calls in this test are designed to demonstrate the keying behavior and throwing the ancillary rp preload here is going to maybe be confusing in the future. There is probably already a more generic test acutally that you can just add referrer policy to and just modify the assertions

Reply:
I created a new test in [24e2703](https://github.com/facebook/react/pull/27096/commits/24e2703c44e6b4de69944818b158897f3b9b08e8)